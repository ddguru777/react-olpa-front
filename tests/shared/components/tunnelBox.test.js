/**
 * Copyright (c) 2015-present, CWB SAS
 *
 * This source code is licensed under the GPL v2.0+ license found in the
 * LICENSE file in the root directory of this source tree.
 */
import React from "react";
import renderer from "react-test-renderer";
import TunnelBox from "shared/components/tunnelBox";

describe("components/tunnelBox", () => {
  it("renders correctly", () => {
    const param = {
      active: { foo: true, bar: false },
      providers: { 1: "foo", 2: "bar", 3: "foobar" },
    };
    const component = renderer.create(
      <TunnelBox params={param} onChange={jest.fn()} />,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
