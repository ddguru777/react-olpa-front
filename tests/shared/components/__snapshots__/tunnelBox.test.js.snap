// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/actionsList renders correctly 1`] = `
<div
  className="mrb-sublist"
>
  <div
    className="mrb-subheader"
  >
    <i
      className="mrb-subheader-right material-icons"
      disabled={false}
      onClick={[Function]}
      onKeyUp={[Function]}
      role="button"
      tabIndex="0"
    >
      keyboard_arrow_up
    </i>
    <div />
  </div>
  <div
    style={
      Object {
        "display": "block",
      }
    }
  >
    <ul
      className="mdc-list"
    />
    <li
      className="selectableListItem onFocusAction mdl-list_action mdc-list-item"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="menuitem"
      style={
        Object {
          "color": "rgb(213, 0, 0)",
          "padding": "0px 16px",
        }
      }
    >
      <i
        aria-hidden="true"
        className="mdc-list-item__graphic material-icons"
      >
        chat_bubble_outline
      </i>
      <div
        aria-label="Add an output response"
        className="contenteditable"
        id="ae_content"
        onBlur={[Function]}
        onFocus={[Function]}
        onInput={[Function]}
        onKeyDown={[Function]}
        onKeyPress={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        spellCheck={false}
        tabIndex={0}
      >
        <span />
      </div>
    </li>
  </div>
</div>
`;
